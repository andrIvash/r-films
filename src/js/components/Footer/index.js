import React, { PureComponent } from 'react';
import { Socials } from '../Socials';

export default class Footer extends PureComponent {
  constructor(props) {
    super(props);
    this.state = {
      socials: [{name: 'fb'}, {name: 'tw'}, {name: 'ln'}]
    }
  }

  render() {
    return (
      <footer className='footer app__footer'>
          footer
          <Socials socials={this.state.socials} />
      </footer>
    );
  }
}

