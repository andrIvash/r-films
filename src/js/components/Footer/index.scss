.footer {
  display: flex;
  flex-shrink: 0;
  align-items: center;
  text-align: left;
  min-height: rem(40px);
}