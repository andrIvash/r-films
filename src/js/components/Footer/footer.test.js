import React from 'react';
import Footer from './index';
import renderer from 'react-test-renderer';

jest.mock('../Logo', () => 'logo');


describe('Footer', () => {
  it('should be defined', () => {
    expect(Footer).toBeDefined();
  });
  it('should render correctly', () => {
    const wrapper = renderer.create(
      <Footer /> ).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
