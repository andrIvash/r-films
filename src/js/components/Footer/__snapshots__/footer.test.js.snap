// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<footer
  className="footer app__footer bg-dark"
>
  <div
    className="container"
  >
    <Logo />
  </div>
</footer>
`;
